src/ui/simulator/toolbox/components/datagrid/renderer/mc-playlist.cpp

```diff
@@ -159,9 +159,9 @@ namespace Renderer
                 }
                 case MCPlaylistCol::WEIGHT:
                 {
-                    std::vector<int> yearsWeight = study->parameters.getYearsWeight();
+                    std::vector<float> yearsWeight = study->parameters.getYearsWeight();
                     assert(y < yearsWeight.size());
-                    return wxString::Format(wxT("%i"), yearsWeight[y]);
+                    return wxString::Format(wxT("%.3f"), yearsWeight[y]);
```

Comment:
Maybe we do not need that precision : 3 significant digits are enough.
2 digits after decimal points should be ok